src/extract-major-beta-version.py

```diff
@@ -0,0 +1,93 @@
+#!/usr/bin/env python3
+
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/
+
+
+import os, re, sys
+
+from github import Github, InputGitAuthor, enable_console_debug_logging
+
+
+#
+# All these are lifted from relbot/utils.py
+#
+
+
+def major_version_from_release_branch_name(branch_name):
+    if matches := re.match(r"^releases[_/]v(\d+)\.0\.0$", branch_name):
+        return int(matches[1])
+    raise Exception(f"Unexpected release branch name: {branch_name}")
+
+
+def get_release_branches(repo):
+    return [branch.name for branch in repo.get_branches()
+            if re.match(r"^releases[_/]v\d+\.0\.0$", branch.name)]
+
+
+def get_latest_beta_major_version(repo):
```

Comment:
Does this get the latest major version number, regardless of whether it's `beta` ?

Reply:
Indeed. 
Will update, thank you!